castai/add-castai-removal-disabled/02-manifests.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+apply:
+- file: patched01.yaml
+  shouldFail: false
+- file: patched02.yaml
+  shouldFail: false
```

Comment:
You should be applying resources which have not already been mutated so you can assert, in a later step, that they have been.

Reply:
Updated, that makes more sense.